app/web/src/newhotness/Map.vue

```diff
@@ -78,7 +78,7 @@
 
     <ComponentContextMenu
       ref="componentContextMenuRef"
-      :viewId="viewId"
+      :viewId="explore.viewId.value"
```

Comment:
This pr removes the view Id prop from this component, should this attr passing here just be removed?

Reply:
Probably should be removed in Explore.vue too since the component is also there

Reply:
I caught it there! but mindlessly kept it here. Removed! Good catch